### ISSUE TITLE
Bug: returning text for internal message should not broadcast the reply.

### ISSUE THREAD

**Issue by hiway** [zentropi/python-zentropi#22]
Example:

`SlackAgent` sends the message internally.

`self.message(name=slack_event['text'], data=slack_event, internal=True)`

And we subclass it like this:

```
class MySlackAgent(SlackAgent):
    @on_message('help')
    def help(self, message):
        return 'This help text is not so helpful.'
```

Expected:
The reply 'This help text is not so helpful.' should not be broadcast, but trigger only internal handlers.

What happens:
The message stays internal, reply is broadcast.

**Comment by hiway** [comment]
Alternative approach: do not trigger internal handlers for messages and requests, since it is more dangerous than useful.

**Comment by hiway** [comment]
Yikes, did not mean to close this issue; the bug is valid and needs to be fixed. 
Not triggering handlers for messages created by the agent itself does not fix it.

**Comment by hiway** [comment]
Fixed in 526dea6fc9f67133a5c95961f9cc8b90fbb07f37.
